### PR TITLE
Web Survey on 20211222

### LINKED ISSUE #3712: thunderbird: security update to 91.4.1
### CVE IDs

CVE-2021-4126, CVE-2021-44538

### Other security advisory IDs

[MFSA2021-55](https://www.mozilla.org/en-US/security/advisories/mfsa2021-55/)

### Description

See [MFSA2021-55](https://www.mozilla.org/en-US/security/advisories/mfsa2021-55/).

### Patches

N/A

### PoC(s)

N/A

### LINKED ISSUE #3713: palemoon: security update to 29.4.3
### CVE IDs

CVE-2021-38503, CVE-2021-38508, CVE-2021-43536, CVE-2021-43537, CVE-2021-43541, CVE-2021-43542, CVE-2021-43545

### Other security advisory IDs

N/A

### Description

See [Pale Moon: Release notes](https://www.palemoon.org/releasenotes.shtml).

### Patches

N/A

### PoC(s)

N/A

### LINKED ISSUE #3715: httpd: security update to 2.4.52
### CVE IDs

CVE-2021-44224, CVE-2021-44790

### Other security advisory IDs

N/A

### Description

```

  *) SECURITY: CVE-2021-44790: Possible buffer overflow when parsing
     multipart content in mod_lua of Apache HTTP Server 2.4.51 and
     earlier (cve.mitre.org)
     A carefully crafted request body can cause a buffer overflow in
     the mod_lua multipart parser (r:parsebody() called from Lua
     scripts).
     The Apache httpd team is not aware of an exploit for the
     vulnerabilty though it might be possible to craft one.
     This issue affects Apache HTTP Server 2.4.51 and earlier.
     Credits: Chamal

  *) SECURITY: CVE-2021-44224: Possible NULL dereference or SSRF in
     forward proxy configurations in Apache HTTP Server 2.4.51 and
     earlier (cve.mitre.org)
     A crafted URI sent to httpd configured as a forward proxy
     (ProxyRequests on) can cause a crash (NULL pointer dereference)
     or, for configurations mixing forward and reverse proxy
     declarations, can allow for requests to be directed to a
     declared Unix Domain Socket endpoint (Server Side Request
     Forgery).
     This issue affects Apache HTTP Server 2.4.7 up to 2.4.51
     (included).
     Credits: æ¼‚äº®é¼ 
     TengMA(@Te3t123)
```

### Patches

N/A

### PoC(s)

N/A

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Survey for updates in `extra-web`, update where available.

Package(s) Affected
-------------------

- `axel` v2.17.11
- `charset-normalizer` v2.0.9 (supersedes, co-authored by #3568)
- `curl` v7.80.0
- `deluge` v2.0.5
- `dropbox` v138.4.2392
- `firefox` v95.0.2
- `httpd` v2.4.52
- `hugo` v0.91.0
- `i2pd` v2.40.0
- `liferea` v1.13.7
- `nextcloud-client` v3.4.1
- `nginx` v1.21.4
- `opera` v82.0.4227.43
- `owncloud-client` v2.9.2.6206
- `palemoon` v29.4.3
- `proton-bridge` v2.0.1
- `qbittorrent` v4.3.9
- `qownnotes` v21.12.4
- `qtox` v1.17.4
- `qutebrowser` v2.4.0
- `range-v3` v0.11.0
- `requests` v2.26.0
- `rclone` v1.57.0
- `seamonkey` v2.53.10.1
- `srain` v1.3.1
- `telegram-desktop` v3.3.0
- `thunderbird` v91.4.1
- `toxcore` v0.2.13
- `vivaldi` v5.0.2497.32
- `weechat` v3.4

Security Update?
----------------

Yes, fixes #3712, #3713, #3715.

Build Order
-----------

```
extra-web/axel
extra-web/curl
extra-web/deluge
extra-web/dropbox
extra-web/firefox
extra-web/httpd
extra-web/hugo
extra-web/i2pd
extra-web/liferea
extra-web/nextcloud-client
extra-web/nginx
extra-web/opera
extra-web/owncloud-client
extra-web/palemoon
extra-web/proton-bridge
extra-web/qbittorrent
extra-web/qownnotes
extra-web/qtox
extra-web/qutebrowser
extra-web/rclone
extra-web/seamonkey
extra-python/charset-normalizer
extra-python/requests
extra-web/srain
extra-libs/range-v3
extra-web/telegram-desktop
extra-web/thunderbird
extra-web/toxcore
extra-web/vivaldi
extra-web/weechat
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by MingcongBai** [comment]
AMD64: Firefox, Thunderbird, Telegram Desktop tested OK.

**Comment by MingcongBai** [comment]
Ready for review.

File: extra-web/telegram-desktop/autobuild/defines [hunk 1]
```diff
@@ -19,8 +19,7 @@ CMAKE_AFTER="-DTDESKTOP_API_TEST=ON \
              -DTDESKTOP_LAUNCHER_BASENAME=telegramdesktop \
              -DTDESKTOP_DISABLE_GTK_INTEGRATION=ON \
              -DDESKTOP_APP_DISABLE_WEBRTC_INTEGRATION=OFF \
-             -DDESKTOP_APP_USE_PACKAGE=ON"
+             -DDESKTOP_APP_USE_PACKAGE=ON \
+             -DDESKTOP_APP_QT6=OFF"
 
 ABTYPE=cmakeninja
```
Comment (WhiredPlanck) [inline extra-web/telegram-desktop/autobuild/defines:25]: Maybe we could unmark this since `tdesktop` only use CMake build system?

**Review by WhiredPlanck — COMMENTED** [review]
Build script LGTM.

**Review by WhiredPlanck — APPROVED** [review]

**Pull request merged**